_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+                <span class="form-input-label">Text messages on my mobile phone</span>
+            </label>
+            <div class="helper-text" id="updates_mobile_text_01_helper">Rates apply. Pelase consult your plan.</div>
+        </div>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_email_01">
+                <input type="radio" class="form-input input-radio" name="updates_notifications_01" id="updates_email_01" value="Email messages" aria-describedby="updates_email_01_helper">
+                <span class="form-input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="helper-text" id="updates_email_01_helper">We will never spam you. Pinky swear.</div>
+        </div>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_email_04">
+                <input type="radio" class="form-input input-radio" name="updates_notifications_01" id="updates_email_03" value="Email messages" disabled>
+                <span class="form-input-label">Disabled radio button</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="field-group form-group">
+        <legend class="field-group-hd form-label">Enhanced radio buttons</legend>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_mobile_text_02">
+                <input type="radio" class="form-input input-radio replace-radio" name="updates_notifications_02" id="updates_mobile_text_02" value="Mobile text messages" aria-describedby="updates_mobile_text_02_helper" checked="checked">
+                <span class="form-input-label">Text messages on my mobile phone</span>
+            </label>
+            <div class="helper-text" id="updates_mobile_text_02_helper">Rates apply. Pelase consult your plan.</div>
+        </div>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_email_02">
+                <input type="radio" class="form-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_02" value="Email messages" aria-describedby="updates_email_02_helper">
+                <span class="form-input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="helper-text" id="updates_email_02_helper">We will never spam you. Pinky swear.</div>
+        </div>
+        <div class="field">
+            <label class="form-label label-inline" for="updates_email_04">
+                <input type="radio" class="form-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_04" value="Email messages" disabled>
+                <span class="form-input-label">Disabled enhanced radio button</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Checkboxes</h4>
+<div class="example-set">
+    <fieldset class="field-group form-group">
```

Comment:
Please revise these field values so that they're a cohesive but general example you'd find in the wild.